spec.html

```diff
@@ -35834,6 +36901,46 @@ <h1>DataView.prototype.constructor</h1>
         <p>The initial value of `DataView.prototype.constructor` is %DataView%.</p>
       </emu-clause>
 
+      <emu-clause id="sec-dataview.prototype.getbigint64">
+        <h1>DataView.prototype.getBigInt64 ( _byteOffset_ [ , _littleEndian_ ] )</h1>
+        <p>When the `getBigInt64` method is called with argument _byteOffset_ and optional argument _littleEndian_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _v_ be the *this* value.
+          1. If _littleEndian_ is not present, let _littleEndian_ be *undefined*.
```

Comment:
This line seems unnecessary, since all unprovided arguments are implicitly undefined?
```suggestion
```

(and throughout)